packages/amplify-graphql-transformer-core/src/transformer-context/index.ts

```diff
@@ -98,4 +101,12 @@ export class TransformerContext implements TransformerContextProvider {
   public isProjectUsingDataStore(): boolean {
     return !!this.resolverConfig?.project || !!this.resolverConfig?.models;
   }
+
+  public isProjectUsingCPK = (): boolean => {
```

Comment:
Can we add unit tests for this check? Thanks!!

Reply:
Looks like we're only mocking the return value for this in the unit tests - thoughts on adding testing for this function?